pennylane/templates/__init__.py

```diff
@@ -15,9 +15,9 @@
 This module contains templates, which are pre-coded routines that can be used in a quantum node.
 """
 
+from .broadcast import *
```

Comment:
You might also have to modify the `from pennylane.templates import template` line in `pennylane/__init__.py` to `from pennylane.templates import template, broadcast` to make `qml.broadcast` work.

Reply:
Right now `broadcast` needs to be imported

Reply:
If I do that we can import ``broadcast`` like ``qml.broadcast``, right? 

I would rather enforce ``qml.templates.broadcast``, so the user knows WHAT gets broadcasted here :)

What do you think?

Reply:
Yep!

Sure, sounds good to me :) Was suggesting `qml.broadcast` as it was mentioned in previous PRs, but `qml.templates.broadcast` is also cool

Reply:
Note that `broadcast` acts on both operations _and_ templates, akin to `qml.inv()` (which I think is also defined in template utils?). So `qml.broadcast` seems more natural to me.

Reply:
You two convinced me. Shall we still keep it physically in the templates module, but just add the import structure @antalszava suggested? Or shall I move it into ``pennylane.utils`` ?

Reply:
Where does inv live? It should probably go in the same place.

(Train of thought: I quite dislike utils - it has become a catch all module, and leads to a lot of circular import issues)

Reply:
That's bad - ``inv()`` lives in utils :)

We can leave it in ``templates`` for now, since changing the location will not be feature breaking if we allow ``qml.broadcast`` import.